File: app/src/main/java/com/mfsoftware/home/api/SignInResponse.java
package com.mfsoftware.home.api;

import com.google.gson.annotations.SerializedName;

public class SignInResponse {
    @SerializedName("token")
    private String token;

    @SerializedName("username")
    private String userName;

    @SerializedName("firstname")
    private String firstName;

    public String getFirstName() {
        return firstName;
    }

    public String getUserName() {
        return userName;
    }

    public String getToken() {
        return token;
    }
}

File: app/src/main/java/com/mfsoftware/home/api/SignInResponse.kt
package com.mfsoftware.home.api

import com.google.gson.annotations.SerializedName

class SignInResponse {
    @SerializedName("token")
    lateinit var token: String

    @SerializedName("username")
    lateinit var userName: String

    @SerializedName("firstname")
    lateinit var firstName: String
}